opm/simulators/flow/FlowMainEbos.hpp

```diff
@@ -83,6 +83,17 @@ namespace Opm
         // Read the command line parameters. Throws an exception if something goes wrong.
         static int setupParameters_(int argc, char** argv)
         {
+            using ParamsMeta = typename GET_PROP(TypeTag, ParameterMetaData);
```

Comment:
Just out of curiosity, does `GET_PROP` not use `typename` internally?

Reply:
Looks like it does not. It's even documented:
```
/*!
 * \ingroup Properties
 * \brief Retrieve a property for a type tag.
 *
 * If you use \c GET_PROP within a template and want to refer to some
 * type (including the property itself), \c GET_PROP must be preceeded by
 * the '\c typename' keyword.
 */
#define GET_PROP(TypeTag, PropTagName) \
    ::Opm::Properties::GetProperty<TypeTag, PTAG(PropTagName)>::p
```

Reply:
> Looks like it does not. It's even documented:

Interesting&mdash;thanks for looking up the details.